src/main/java/com/tabgok/entity/Machine.java

```diff
@@ -0,0 +1,54 @@
+package com.tabgok.entity;
+
+import com.tabgok.data.StringData;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * The machine is a model of a machine.  Importantly, the machine is the top-level
+ * model for a machine, meaning it contains within it either references to, or values of,
+ * everything within a machine from Hardware to Filesystems to applications.
+ */
+public class Machine extends Entity {
+    private StringData MachineID;
+    private final HashSet<String> mountedFilesystems = new HashSet<>();
+    
+    private Machine(){}
+    
+    public static Machine getMachine(){
```

Comment:
"getMachine()" sounds to me like there's just one. simply ".create()" is pretty conventional for something like this.